citrination_client/client.py

```diff
@@ -55,6 +63,94 @@ def search(self, pif_query):
+        :param property_units: One or more strings with the property units to match.
+        :param reference_doi: One or more strings with the DOI to match.
+        :param include_datasets: One or more integers with dataset IDs to match.
+        :param exclude_datasets: One or more integers with dataset IDs that must not match.
+        :param from_index: Index of the first record to match.
+        :param size: Total number of records to return.
+        :return: :class:`.PifSearchResult` object with the results of the query.
+        """
+        system_query = SystemQuery()
+        system_query.names = FieldOperation(
+            extract_as='name',
+            filter=[Filter(equal=i) for i in self._get_list(name)])
+        system_query.chemical_formula = ChemicalFieldOperation(
+            extract_as='chemical_formula',
+            filter=[ChemicalFilter(equal=i) for i in self._get_list(chemical_formula)])
+        system_query.references = ReferenceQuery(doi=FieldOperation(
+            extract_as='reference_doi',
+            filter=[Filter(equal=i) for i in self._get_list(reference_doi)]))
+
+        # Generate the parts of the property query
+        property_name_query = FieldOperation(
+            extract_as='property_name',
+            filter=[Filter(equal=i) for i in self._get_list(property_name)])
+        property_units_query = FieldOperation(
+            extract_as='property_units',
+            filter=[Filter(equal=i) for i in self._get_list(property_units)])
+        property_value_query = FieldOperation(
+            extract_as='property_value',
+            filter=[])
+        for i in self._get_list(property_value):
+            property_value_query.filter.append(Filter(equal=i))
+        if property_min is not None or property_max is not None:
+            property_value_query.filter.append(Filter(min=property_min, max=property_max))
+
+        # Generate the full property query
+        system_query.properties = PropertyQuery(
+            name=property_name_query,
+            value=property_value_query,
+            units=property_units_query)
+
+        # Run the query
+        query = PifQuery(
+            system=system_query,
+            from_index=from_index,
+            size=size,
+            score_relevance=True,
+            include_datasets=include_datasets,
+            exclude_datasets=exclude_datasets)
+        return self.search(query)
+
+    @staticmethod
+    def _get_list(values):
+        """
+        Helper method that wraps values in a list. If the input is a list then it is returned. If the input is None
+        then an empty list is returned. For anything else, the input value is wrapped as a single-element list.
+
+        :param values: Value to make sure exists in a list.
+        :return: List with the input values.
+        """
+        if values is None:
+            return []
+        elif isinstance(values, list):
```

Comment:
Is a `tuple` also be a legal input sequence, i.e. `simple_chemical_search(name=('A', 'B'))`